proposals/0013-semantic-index-assignment.md

```diff
@@ -0,0 +1,226 @@
+* Proposal: [0013](0013-semantic-index-assignment.md)
+* Author(s): [Joshua Batista](https://github.com/bob80905)
+* Sponsor: TBD
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8
+* PRs: [#6108](https://github.com/microsoft/DirectXShaderCompiler/pull/6108)
+* Issues:
+  [#5827](https://github.com/microsoft/DirectXShaderCompiler/issues/5827)
+
+## Introduction
+
+Shaders with entry point functions may have semantics attached to the function's parameters.
+This spec specifies how legal semantic usage is checked on entry point shaders. 
+The purpose of the spec is to properly define how diagnostics get emitted in Sema for 
+higher-dimensional parameters with semantics, that later get flattened in SROA. 
+Although the DXIL spec specifies how semantic indices are assigned in 
+https://github.com/Microsoft/DirectXShaderCompiler/blob/main/docs/DXIL.rst#semantic-index-assignment,
+there is no indication of how the indices are used to generate diagnostics.
+So, the compiler will need to take the implementation in SROA for parameter flattening, and
+bring it over to Sema, so that diagnostics are properly emitted there.
+
+## Motivation
+
+Currently, the compiler does not emit any diagnostics for semantic index collisions in Sema.
+Such collisions are detected and diagnosed in SROA, much later. Additionally, due to the complexity
+of SROA, it is difficult to understand, maintain, and extend the code that handles semantic index collision. 
+For example, in https://github.com/microsoft/DirectXShaderCompiler/issues/5827, there was
+no existing infrastructure to check for SV conflicts. For compiler developers, there should already
+exist an infrastructure for adding SV conflict detection based on known semantic indices.
+
+## Proposed solution
+The proposed solution is to copy and perhaps simplify the infrastructure in SROA for semantic
+index collision detection, and move it into Sema. 
+For compiler developers, it would be beneficial to define an infrastructure for semantic index
+collision detection, so that as new semantics get added, it becomes easy to add the necessary logic
+that would prevent conflicts with other attributes, or index collisions of the same attribute.
+Also, by moving this infrastructure to Sema, it can be made cleaner, and more efficient.
+For HLSL developers, moving these diagnostics earlier to Sema will provide more information, and
+the information will be provided sooner, decreasing compile time.
+
+## Detailed design
+
+*Entry function selection (specifics outside scope of this proposal)

```

Comment:
I think you intend these to be formatted as a bullet list?  In which case you need spaces between the `*` and the rest of the text.  As it is now, the preview sticks most of this section onto a single line.